core/src/main/java/org/apache/carbondata/core/datastore/page/encoding/EncodingFactory.java

```diff
@@ -92,6 +90,10 @@ public ColumnPageDecoder createDecoder(List<Encoding> encodings, List<ByteBuffer
       RLEEncoderMeta metadata = new RLEEncoderMeta();
       metadata.readFields(in);
       return new RLECodec().createDecoder(metadata);
+    } else if (encoding == BOOL_BYTE) {
+      BooleanEncoderMeta metadata=new BooleanEncoderMeta();
```

Comment:
Not compliant to coding standard. Add space before and after `=`

Reply:
I think it is better to use RLE on byte, since it just has two values

Reply:
ok, I have changed the encoding from snappy to RLE